File: src_f/tests/helloworld/helloworld.f90
!
! Hello world fortran example.
!

program Helloworld
use utils_time
implicit none

real(Configure_Long_Real) :: tstart, tstop

    tstart = time_DTime()
    print *, "Hello, World!"
    tstop = time_DTime()

    print *, "time = ", tstop - tstart

end program Helloworld

File: src_c/tests/helloworld/helloworld.cpp
#include <stdio.h>

#include "utils_time.h"

int main()
{
    double tstart = 0.0;
    double tstop = 0.0;

    tstart = time_DTime();
    printf("Hello, World!\n");
    tstop = time_DTime();

    printf("time = %le\n", tstop - tstart);

    return 0;
}

